src/profile-logic/profile-data.js

```diff
@@ -1474,6 +1474,27 @@ export function accumulateCounterSamples(
   return accumulatedSamples;
 }
 
+/**
+ * The memory counter contains relative offsets of memory. In order to draw an interesting
+ * graph, take the memory counts, and find the minimum and maximum values, by
+ * accumulating them over the entire profile range. Then, map those values to the
+ * accumulatedCounts array.
```

Comment:
nit: I believe this comment needs to be changed

Reply:
Oh, right. Changed it, thanks!